test/calculator.test.mjs

```diff
@@ -51,94 +51,128 @@ describe('calculator', function () {
+  it('should only allow one operator key to be depressed at a time', function () {
     // write your own tests
-    assert.strictEqual(true, false, 'update this test');
+    button.multiply.click();
+    button.add.click();
+    button.divide.click();
+    button.multiply.click();
+    const depressedOperatorKey = document.querySelectorAll('.is-depressed')
+    //key.classList.add('is-depressed'); // Operator keys should be depressed when they're clicked on
+    assert.strictEqual(depressedOperatorKey.length, 1, 'Only one operator key can be pressed consecutively');
   })
 
-  xit('should show new number if pressed after an operator key', function () {
+  it('should show new number if pressed after an operator key', function () {
     // write your own tests
-    assert.strictEqual(true, false, 'update this test');
+    button.one.click()
+    button.add.click()
+    button.four.click()
+    console.log(display.textContent)
+    assert.strict(display.textContent, 4, 'expected number should equal 4');
   });
 
-  xit('should add two numbers together', function () {
+  it('should add two numbers together', function () {
     // write your own tests
     button.nine.click();
     button.add.click();
     button.eight.click();
     button.calculate.click();
+    console.log(display.textContent)
     assert.equal(display.textContent,  17)
   })
-  xit('should subtract two numbers', function () {
+  it('should subtract two numbers', function () {
     // write your own tests
     button.nine.click();
     button.subtract.click();
     button.eight.click();
     button.calculate.click();
+    console.log(display.textContent)
     assert.equal(display.textContent,  1)
   })
-  xit('should multiply two numbers', function () {
+  it('should multiply two numbers', function () {
     // write your own tests
     button.four.click();
     button.multiply.click();
     button.seven.click();
     button.calculate.click();
+    console.log(display.textContent)
     assert.equal(display.textContent, 28)
   })
-  xit('should divide two numbers', function () {
+  it('should divide two numbers', function () {
     // write your own tests
     button.four.click();
     button.four.click();
     button.divide.click();
     button.one.click();
     button.one.click();
     button.calculate.click();
+    console.log(display.textContent)
     assert.equal(display.textContent, 4)
   })
   // write more tests below. Think about how to test for "unhappy paths".
   // Try to break it in as many ways as you can!
-
-  xit('can chain multiple operations together', function () {
+  xit('can only add one decimal at a time', function () {
+    button.four.click();
+    button.decimal.click();
+    button.decimal.click();
+    const depressedDecimal = document.querySelector('is-depressed');
+    assert.strictEqual(depressedDecimal.length, 1, 'only 1 decimal can be clicked')
   });
-
+  xit('can only calculate up to a certain number limit', function () {
```

Comment:
I'm not sure what you are trying to test. Is the limit you mean here, the limit of characters that can fit into the display?

Reply:
Yes I want to limit the number of characters that show in the display like a actual calculator. I may be thinking more into it if this was a full calculator but when it reaches a certain limit the calculator would be, ex 9999999e^17. But that may require more coding logic to the calculator.